exonum/src/crypto.rs

```diff
@@ -612,7 +612,7 @@ impl CryptoHash for i64 {
 
 impl CryptoHash for () {
     fn hash(&self) -> Hash {
-        Hash::zero()
+        hash(&[])
```

Comment:
I wonder if it could make sense to return a constant here. Is this doable without `const fn` support?

Reply:
I suppose it's not possible without things like `lazy_static`.

Reply:
>I suppose it's not possible without things like lazy_static.

I'd say that transmuting an `[u8; 32]` would be a nice solution

Reply:
Provided that we add a tests that checks that the the result of transmute is indeed equal to hash(&[]).